p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -4876,22 +4877,18 @@ match_kind {
 ~ End P4Example
 
 These identifiers correspond to the P4~14~ match kinds with the same
-names. The semantics of these annotations is actually irrelevant for
-describing the behavior of the P4 abstract machine; how they are used
-influences only the control-plane API and the actual implementation of
-the look-up table. From the point of view of the P4 program, a look-up
-table is an abstract finite map that is given a key and produces as a
-result either an action or a "miss" indication, as described in
-Section [#sec-mau-semantics].
+names. From the point of view of the P4 program, a look-up table is an
+abstract finite map that is given a key and produces as a result
+either an action or a "miss" indication, as described in Section
+[#sec-mau-semantics].
 
 If a table has no ```key``` property, then it contains no look-up
-table, just a default action, which is always executed (i.e., the
```

Comment:
The text is about a table without a key, not a table with an empty key.
The table can still have lots of actions in the actions list.